scripts/scil_compute_todi.py

```diff
@@ -33,39 +35,39 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
                                 formatter_class=argparse.RawTextHelpFormatter)
 
-    p.add_argument('in_bundle',
+    p.add_argument('input_tracto',
```

Comment:
Do you have any use case where it is not a bundle ? Moreover, the standard in scilpy is in_. Can you change it ?

Reply:
It can be used for any tractogram, similar to a TDI map or image